graph.ts

```diff
@@ -1,12 +1,8 @@
 import { View, parse } from 'vega';
 import { compile } from 'vega-lite';
-import SVGO from 'svgo';
-
-const svgo = new SVGO({
-  js2svg: {
-    pretty: true,
-  },
-});
+// eslint-disable-next-line @typescript-eslint/ban-ts-comment
+// @ts-ignore
+import { optimize } from 'svgo';
```

Comment:
Sorry to ignore the compiler and linter here, absolutely hate doing that. Seemed like a waste of time to temporarily overwrite their typings when the [contributors on that repo are in the process of adding TS support](https://github.com/svg/svgo/pull/1370).